src/com/form.js

```diff
@@ -266,31 +261,27 @@
 				});
 			}
 		};
-		
+
 		return {
 			init: init,
 			enable: enable,
-			validate: validate,
 			reset: reset,
 			preview: preview,
 			post: post,
-			submitting: submitting,
```

Comment:
This think we need to keep that one for the captcha...

Reply:
A submitting state is added on the form so it will work with recaptcha or not. This was used to execute the submitting method on the form-field which does not make sense. (The form is submitting, not the fields itself)